content/js/index.md

```diff
@@ -12,6 +12,7 @@ application at the Skygear [Developer Portal](https://portal.skygear.io)
 website. After you registered, go to the **INFO** tab and copy down your
 `Server EndPoint` and `API Key`.
 
+![Example Image](/content/common/portal.png)
```

Comment:
The file name should be more specific. Imagine there can be more screenshots of the portal in other parts of the doc